gateway/ln-gateway/src/lib.rs

```diff
@@ -984,20 +985,28 @@ impl Gateway {
 
     /// Requests the gateway to pay an outgoing LN invoice using its own funds.
     /// Returns the payment hash's preimage on success.
-    async fn handle_pay_invoice_self_msg(
+    async fn handle_pay_invoice_for_operator_msg(
         &self,
-        payload: PayInvoicePayload,
+        payload: PayInvoiceForOperatorPayload,
     ) -> AdminResult<Preimage> {
         let GatewayState::Running { lightning_context } = self.get_state().await else {
             return Err(AdminGatewayError::Lightning(
                 LightningRpcError::FailedToConnect,
             ));
         };
 
+        let max_fee = 50
```

Comment:
Can we make this a constant with an explanation for why 50 was chosen?

Reply:
Its just the default of ldk, 1 % plus 50 sats.

Reply:
Personally, I would not choose a constant here as the value is not critical and it makes the code less localized.

Reply:
You can make the constant in the same function so that its not reused. I don't usually like directly inlining numbers in code because (usually after some time), people forget why it was there in the first place and it just becomes a magic number.

Reply:
Done.